### PR TITLE
Reviewed docker file and gh actions ci

### PR DESCRIPTION
**Pull request by ysibirski**
Final new version of the PR.

One note here. {{secrets.ENV}} stopped working for some reason under bash scripts. I could not find the reason for it, will try to run it on Monday again but {{vars.ENV}} works just fine :shrug:

### CODE REVIEW

File: ci/setup_env.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+export PROD_RELEASE="false"
+export DEV_RELEASE="false"
+export TEST_RELEASE="false"
+export COMMON_FILE_LOCATION='ci/common.sh'
+
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+github_tag="${GITHUB_REF_NAME:-}"
+release_branch="${RELEASE_BRANCH:-release}"
+prod_release_regex='^[0-9]+\.[0-9]+\.[0-9]+$'
+dev_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+){1}$'
```
Comment (JackPiri) [inline ci/setup_env.sh:14]: In previous PR it seems to me this was fixed with:
`dev_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+){1}$'`

**Review by JackPiri — CHANGES_REQUESTED** [review]
The PR is almost fine. Please just address comment related to `dev_release_regex` (I remember it was fine during my last review on the other PR) and then I'll approve.
Thanks

File: ci/setup_env.sh (hunk 1 above)
Comment (ysibirski) [reply ci/setup_env.sh:14]: :thinking: from out last conversation with @lander86 it seems like the decision was made towards removing `.` after rc :shrug:

Comment (JackPiri) [reply ci/setup_env.sh:14]: Ok then it's fine to me.
Thanks

**Review by JackPiri — APPROVED** [review]
The PR is fine to me.
Thanks

**Review by lander86 — APPROVED** [review]
LGTM

**Pull request merged**